versions.mk

```diff
@@ -19,8 +19,8 @@
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
 VERSION ?= v24.9.2
 
-GOLANG_VERSION ?= 1.23.5
+GOLANG_VERSION ?= 1.23.6
 
-GOLANGCI_LINT_VERSION ?= v1.63.4
+GOLANGCI_LINT_VERSION ?= v1.64.5
```

Comment:
is there a way we can move this to a place that Dependabot will take care of the bump? maybe a `tools.go` under hack folder?

Reply:
This is [where](https://github.com/NVIDIA/gpu-operator/blob/43ec0296a94d2c73aa5030a7ae31afc9c5cbdd08/.github/workflows/ci.yaml#L87) we ultimately consume the version. Is it possible to have dependabot bump the version here?